src/test/java/org/javarosa/xform/parse/ExternalSecondaryInstanceTest.java

```diff
@@ -0,0 +1,49 @@
+package org.javarosa.xform.parse;
+
+import org.javarosa.core.model.FormDef;
+import org.javarosa.core.model.condition.EvaluationContext;
+import org.javarosa.core.model.condition.IFunctionHandler;
+import org.javarosa.core.test.FormParseInit;
+import org.junit.Test;
+
+import java.nio.file.Path;
+import java.util.List;
+
+import static org.javarosa.core.reference.ReferenceManagerTestUtils.setUpSimpleReferenceManager;
+import static org.javarosa.test.utils.ResourcePathHelper.r;
+
+public class ExternalSecondaryInstanceTest {
+    @Test
+    public void externalInstanceDeclaration_ShouldBeIgnored_WhenNotReferenced() {
+        Path formPath = r("minimal-pulldata.xml");
```

Comment:
For this test it doesn't matter that the form uses `pulldata`, right? If I'm correct, I'd suggest removing it to make it easier to understand.